### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This is a series of spelling fixes to Scumblr (including one for misspelling the product name...).

This PR can, of course, be folded together. However, initially, it is much easier to rebase and discuss individual concepts as individual commits. It's also possible to split this PR into a couple, one for API changes and one for other things, but there aren't many such items in this PR and I'd rather not spend that time on it.

### CODE REVIEW

**Comment by sbehrens** [comment]
Hi Jsoref, 

We've done a bunch of code changes since the 1.0 release.  Any way you could run your spellchecker against the new code as well?

**Comment by jsoref** [comment]
@sbehrens: Here's a set of fixes for things which were visibly misspelled as identified in insertions/removals between when I branched off.

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Use this hook to configure devise mailer, warden hooks and so forth.
 # Many of these configuration options can be set straight in your model.
 Devise.setup do |config|
-  #allows devise to use mutal TLS for auth
+  #allows devise to use mutual TLS for auth
```
Comment (jsoref) [inline config/initializers/devise.rb]: I'm not sure if this is the correct fix -- I don't have enough context.

File: lib/scumblr_tasks/security/github_analyzer.rb [hunk 2]
```diff
@@ -386,13 +386,13 @@ def run
           end
 
           if response.nil?
-            @retry_interavl = 0
+            @retry_interval = 0
```
Comment (jsoref) [inline lib/scumblr_tasks/security/github_analyzer.rb]: This series `@retry_interavl` is an actual bug as opposed to just a typo in a comment or a consistently misspelled api.

**Comment by sbehrens** [comment]
Nice, thanks again.  Would be interested in you checking some of our other security tools using your spelling scripts: https://github.com/netflix/sketchy and https://github.com/netflix/sleepy-puppy if you are willing to help!

**Comment by jsoref** [comment]
I can, but at some point, you're going to need to update lodash. They misspelled `forty` (it seems to be a common error, I remember some other project recently making the same mistake..) -- and I haven't hit their project yet...

https://github.com/Netflix/sketchy/pull/37

**Comment by jsoref** [comment]
Thanks for taking these PRs. FWIW, this output is manual, I have a script which lists tokens that it doesn't recognize as words, and then it's up to me to decide based on what's left which things are typos and which are ok. (And I'm not perfect.)

Some stats:

Tokens | Selected | Words | Repo/PRs
------ | ----------------- | ----- | --------
275 | 21 | 19 | Netflix/sketchy#37
1058 | 39 | 35 | Netflix/Scumblr#107 Netflix/Scumblr#109
4816 | 26 | 20 | Netflix/sleepy-puppy#11

Because everything after the first column is manual (I do have a tool to sort the PRs alphabetically, although it looks like I didn't necessarily run it for some of these), it isn't a guarantee there aren't other errors. -- While trying to build this table, I found two that I had missed.

**Pull request merged**